theme6/source/javascripts/featured_products.js

```diff
@@ -0,0 +1,26 @@
+var images = ["bear_toy", "train_toy", "plane_toy", "ship_toy"];
+var i = 0;
+
+const leftArrow = document.getElementById("leftArrow");
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).